samples/apps/copilot-chat-app/webapp/src/libs/auth/AuthHelper.ts

```diff
@@ -54,14 +54,14 @@ const logoutRequest: EndSessionRequest = {
 };
 
 const ssoSilentRequest = async (msalInstance: IPublicClientApplication) => {
-    await msalInstance.ssoSilent({ account: msalInstance.getAllAccounts()[0], scopes: Constants.msal.semanticKernelScopes });
+    await msalInstance.ssoSilent({ account: msalInstance.getActiveAccount() || undefined, scopes: Constants.msal.semanticKernelScopes });
 };
 
 const loginAsync = async (instance: IPublicClientApplication) => {
     if (Constants.msal.method === 'redirect') {
-        await instance.loginRedirect({ account: instance.getAllAccounts()[0], scopes: Constants.msal.semanticKernelScopes });
+        await instance.loginRedirect({ account: instance.getActiveAccount() || undefined, scopes: Constants.msal.semanticKernelScopes });
```

Comment:
I don't think it makes sense to pass account here or in loginPopup since we're not setting the active account until the redirect completes. 

Let's remove account from these calls and bypass the user hint altogether on log in (but keep it for the token sso) https://learn.microsoft.com/en-us/azure/active-directory/develop/msal-js-sso#without-user-hint